### PR TITLE
Drew.auditupdates: Upgrade node, remove delegation, fix identity module

### PR DESCRIPTION
**Pull request by drewstone**
Upgrades the node with all the new substrate features.

### CODE REVIEW

**Comment by drewstone** [comment]
@jnaviask I could use a code review on this. It's ready I believe.

**Comment by jnaviask** [comment]
In setup.sh, it calls to build.sh, but build.sh is now in the scripts directory, so just need to fix the path on that.

File: modules/edge-identity/src/identity.rs [hunk 1]
```diff
@@ -57,316 +61,338 @@ type BalanceOf<T> = <<T as Trait>::Currency as Currency<<T as system::Trait>::Ac
...
+            // Grab record
+            let record = <IdentityOf<T>>::get(&identity_hash).ok_or("Identity does not exist")?;
+            // Ensure the record is not verified
+            ensure!(record.stage != IdentityStage::Verified, "Already verified");
+            // Ensure the record isn't expired if it still exists
+            ensure!(<system::Module<T>>::block_number() <= record.expiration_length, "Identity expired");
+            // Check that original sender and current sender match
+            ensure!(record.account == _sender, "Stored identity does not match sender");
+            // Check that the original sender and the attestation sender match
+            // TODO
```
[the remark this answers is not in the document]
Comment (drewstone) [reply modules/edge-identity/src/identity.rs]: It seems that it's already implemented above it (comments are nearly identical)

File: node/service/src/chain_spec.rs [hunk 2]
```diff
@@ -32,461 +35,477 @@ const DEFAULT_PROTOCOL_ID: &str = "edg";
...
 pub fn cwci_testnet_genesis(
-	initial_authorities: Vec<(AccountId, AccountId, AuthorityId)>,
-	root_key: AccountId,
-	endowed_accounts: Option<Vec<AccountId>>,
-	initial_verifiers: Option<Vec<AccountId>>,
+    initial_authorities: Vec<(AccountId, AccountId, AuraId, GrandpaId)>,
+    root_key: AccountId,
+    endowed_accounts: Option<Vec<AccountId>>,
+    initial_verifiers: Option<Vec<AccountId>>,
 ) -> GenesisConfig {
```
[the remark this answers is not in the document]
Comment (drewstone) [reply node/service/src/chain_spec.rs]: Gotcha, I'll make the cwci configuration parameters much shorter.

**Pull request merged**